### PR TITLE
feat: Improve style on tabs

### PR DESCRIPTION
**Pull request by Gara64**
The tabs display was kind of broken on the lateral view on firefox. See before/after:
Capture d’écran du 2019-12-11 11-30-01
Capture d’écran du 2019-12-11 11-41-52

### CODE REVIEW

File: src/popup/scss/base.scss [hunk 1]
```diff
@@ -302,14 +302,12 @@ header {
...
-                @include themify($themes) {
-                    color: themed('mutedColor');
-                }
+                color: $text-muted;
 
                 &:hover, &:focus {
                     @include themify($themes) {
                         background-color: themed('tabBackgroundHoverColor');
+                        color: themed('primaryColor');
                     }
```
Comment (Crash--) [inline src/popup/scss/base.scss]: Why do we need those 2 changes?

Comment (Gara64) [reply src/popup/scss/base.scss]: Turns out it is actually not necessary, I'll remove it

File: src/popup/scss/misc.scss [hunk 2]
```diff
@@ -225,6 +225,9 @@ app-vault-icon {
 
 .icon-cozy-inline {
     color: $gray-light;
+    display: flex;
+    justify-content: center;
+    margin: auto;
```
Comment (y-lohse) [inline src/popup/scss/misc.scss]: What is the margin definition for?

Comment (Gara64) [reply src/popup/scss/misc.scss]: It seemed useful at first, but is no longer necessary. I'll remove it

**Pull request merged**